### ISSUE TITLE
Not selecting at Click

### ISSUE THREAD

**Issue by rskinnerc** [fritx/vue-at#32]
Hi.

I'm using vue-at textarea which works fine until the point where I click an item from the showed up list. In that moment, the clicked item does not get selected. Any idea why it could be happening?

**Comment by fritx** [comment]
@VetDev what was your operating system and browser?

**Comment by fritx** [comment]
https://fritx.github.io/vue-at/#/en/textarea

**Comment by iadambrown** [comment]
For me, it seems to be when using v-model to bind a value to the textarea (e.g. `v-model="body"`. Switching this to `v-text=body` allows selection of items from the list which then appear in the textarea, _however_ when saving the form that the textarea is in, the `body` value hasn't been updated since it's no longer bound to the textarea...

**Comment by iadambrown** [comment]
FYI the demo above does work for me - is it using `v-model` or `v-text`? Or just a hard-coded value within the textarea?

**Comment by iadambrown** [comment]
Just saw in another issue that `v-model` isn't supported yet (https://github.com/fritx/vue-at/issues/29#issuecomment-349206886) :)

**Comment by fritx** [comment]
@iadambrown @VetDev yeah, `v-model` is not supported yet,
but I'm very interested in landing it now: `<at v-model="text"></a>`??

**Comment by iadambrown** [comment]
If you can get v-model working, that would be great! I switched to at.js (temporarily) solely because of that one drawback, though it feels gross to need jquery as a dep, just for that one library... `<at v-model="body"></at>` would be very, very welcome in my current project ;)

**Comment by jakedevsquad** [comment]
Has this issue with clicking been resolved yet? I am using the 2.4.0 beta with v-model support and clicking does not select the item... Any help would be great!

**Comment by fritx** [comment]
@jakedevsquad @iadambrown `v-model` had been supported, and you should follow the docs usage like https://fritx.github.io/vue-at/#/en/vmodel

```vue
<at v-model="html">
  <div contenteditable></div>
</at>

<at-ta v-model="text">
  <textarea></textarea>
</at-ta>
```
